rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -4276,29 +4261,23 @@ void opcode_0182(const ScriptArguments& args, const ScriptContact arg1, const Sc
 
     opcode 0183
     @arg player Player
-    @arg arg2 
+    @arg value 
 */
-bool opcode_0183(const ScriptArguments& args, const ScriptPlayer player, const ScriptInt arg2) {
-    RW_UNIMPLEMENTED_OPCODE(0x0183);
-    RW_UNUSED(player);
-    RW_UNUSED(arg2);
+bool opcode_0183(const ScriptArguments& args, const ScriptPlayer player, const ScriptInt value) {
     RW_UNUSED(args);
-    return false;
+    return player->getCharacter()->getCurrentState().health <= value;
```

Comment:
This opcode is unused, so can't really tell if this correct, but I compared asm instructions, and they seemed the same for ```0183``` ```0184``` ```0185```